dev/benchmarks/macrobenchmarks/lib/src/backdrop_filter.dart

```diff
@@ -0,0 +1,109 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+class BackdropFilterPage extends StatefulWidget {
+  @override
+  _BackdropFilterPageState createState() => _BackdropFilterPageState();
+}
+
+class _BackdropFilterPageState extends State<BackdropFilterPage> with TickerProviderStateMixin {
+  static const int numRows = 15;
+  static const int numCols = 5;
+  bool _blurGroup = false;
+  bool _blurTexts = true;
+  AnimationController animation;
+
+  @override
+  void initState() {
+    super.initState();
+    animation = AnimationController(vsync: this, duration: Duration(seconds: 1));
```

Comment:
```backdrop_filter.dart:20:60 • prefer_const_constructors```

You can open the "Dart Analysis" tool in IntelliJ to discover these kind of warnings locally.

Reply:
Odd, the window shows a green check mark for me.  I actually had a couple of those warnings and fixed them, but they don't show now.